pkg/svcgen/app.go

```diff
@@ -0,0 +1,14 @@
+package svcgen
+
+import (
+	"github.com/izumin5210/grapi/pkg/gencmd"
+	"github.com/izumin5210/grapi/pkg/protoc"
+	"github.com/izumin5210/grapi/pkg/svcgen/params"
+)
+
+type CreateAppFunc func(*gencmd.Ctx, *gencmd.Command) (*App, error)
+
+type App struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type App should have comment or be unexported